some-art/UIImageView+load.swift

```diff
@@ -11,13 +11,12 @@ import UIKit
 
 extension UIImageView {
     func load(url: URL) {
-        DispatchQueue.global().async { [weak self] in
-            if let data = try? Data(contentsOf: url) {
-                if let image = UIImage(data: data) {
-                    DispatchQueue.main.async {
-                        self?.image = image
-                    }
-                }
+        DispatchQueue.global(qos: .userInteractive).async { [weak self] in
```

Comment:
thought the "user interactive" Qos class was appropriate here. referenced this tutorial: https://www.raywenderlich.com/5370-grand-central-dispatch-tutorial-for-swift-4-part-1-2